services/server/src/sourcify-chains-default.json

```diff
@@ -1380,5 +1380,16 @@
         "url": "https://sepolia-blockscout.scroll.io"
       }
     }
+  },
+  "1030": {
+    "sourcifyName": "Conflux eSpace",
+    "supported": true,
+    "etherscanApi": {
```

Comment:
I don't think this explorer is etherscan

Reply:
They have a similar API, so they are fully compatible and have passed the necessary CI-tests.

Reply:
![image](https://github.com/ethereum/sourcify/assets/29679226/559c6b70-d596-4780-8de5-beb8c599e0c7)

Reply:
Ok, I was confused by it. I tried it and it works